webapp/templates/sidebar.html.tmpl

```diff
@@ -26,10 +26,10 @@
   <footer class="footer-side-menu">
     <ul class="footer-list">
       <li class="footer-list-item">
-        <i class="eos-icons" title="" data-html="true" data-toggle="tooltip" data-title="{{ escapedTemplate "license" . }}" data-trigger="hover click">find_in_page</i>
+        <i class="eos-icons" title="" data-html="true" data-toggle="tooltip" data-title="SUSE" data-trigger="hover click">find_in_page</i>
       </li>
       <li class="footer-list-item">
-        <i class="eos-icons" title="" data-html="true" data-toggle="tooltip" data-title="{{ .copyright }}" data-trigger="hover click">info</i>
+        <i class="eos-icons" title="" data-html="true" data-toggle="tooltip" data-title="2020" data-trigger="hover click">info</i>
```

Comment:
another shortcut which I might need to take over next week. I think `copyright` is a var somewhere